guides/source/action_cable_overview.md

```diff
@@ -120,6 +120,17 @@ by this same current user, you're also ensuring that you can later retrieve all
 connections by a given user (and potentially disconnect them all if the user is deleted
 or unauthorized).
 
+If you use Device for authenticaion, you can get `current_user` from warden:
```

Comment:
I think you typod 'Devise' here.

Reply:
Yes. Sorry. A typo I do too often. Fixed.